packages/tabs/addon/components/nucleus-tabs/tab-panel.js

```diff
@@ -0,0 +1,112 @@
+import Component from '@ember/component';
+import { classNames, attributeBindings, classNameBindings, tagName, layout as templateLayout } from '@ember-decorators/component';
+import layout from '../../templates/components/nucleus-tabs/tab-panel';
+import { get, computed } from '@ember/object';
+import defaultProp from '@freshworks/core/utils/default-decorator';
+
+/**
+  __Usage:__
+
+  [Refer component page](/docs/components/nucleus-tabs)
+
+  @class Nucleus Tab Panel
+  @namespace Components
+  @extends Ember.Component
+  @public
+*/
+@tagName('div')
+@templateLayout(layout)
+@classNames('nucleus-tabs__panel')
+@classNameBindings('isActive:is-active')
+@attributeBindings('tabindex')
+@attributeBindings('role')
+@attributeBindings('aria-labelledby')
+class TabPanel extends Component {
+
+  /**
+  * disabled
+  *
+  * @field disabled
+  * @type string
+  * @default 'false'
+  * @readonly
+  * @public
+  */
+  @defaultProp
+  disabled = 'false';
+
+  /**
+  * tabindex
+  *
+  * @field tabindex
+  * @default '0'
+  * @type string
+  * @public
+  */
+  tabindex = '0';
+
+  /**
+  * role
+  *
+  * @field role
+  * @type string
+  * @public
+  */
+  role = 'tabpanel'
+
+  /**
+  * isActive
+  *
+  * @field isActive
+  * @type boolean
+  * @public
+  */
+  @computed('selected', function() {
+    return (get(this, 'selected') === get(this, 'name'));
+  })
+  isActive;
+
+  /**
+  * aria-labelledby
+  *
+  * @field aria-labelledby
+  * @type string
+  * @public
+  */
+  @computed('tabListItems.[]', function() {
+    const tabListItems = get(this, 'tabListItems');
+    const tabList = tabListItems.findBy('name', get(this, 'name'));
+    return (tabList)? tabList.id : '';
+  })
+  'aria-labelledby';
+
+  /**
+  * init 
+  *
+  * @method init
+  * @description lifecycle event
+  * @public
+  *
+  */
+  init() {
+    super.init(...arguments);
```

Comment:
I think this lifecycle hook can be removed.

Reply:
Removed.